packages/vue/src/components/organisms/SfTopBar/SfTopBar.stories.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+
+import SfTopBar from "./SfTopBar.vue";
+
+storiesOf("Organisms|TopBar", module)
+  .add(
+    "Basic",
+    () => ({
+      template: "<sf-top-bar />",
+      components: {
+        SfTopBar
+      }
+    }),
+    {
+      info: {
+        summary: "<p>Component for displaying Topbar</p>"
+      }
+    }
+  )
+  .add(
+    "[slot] left",
+    () => ({
+      template: `
+    <sf-top-bar>
+      <template slot="left">left content</template>
+    </sf-top-bar>
+    `,
+      components: {
+        SfTopBar
+      }
+    }),
+    {
+      info: true
+    }
+  )
+  .add(
+    "[slot] default",
```

Comment:
I guess this is the basic one so there is no need to duplicate it :)

Reply:
This is a little different from the basic, because it customises the center content (default slot). 
What do you think ?

Reply:
Isn't the basic use case filling a default slot?

Reply:
Oh right. This actually wasn't the case. I've done a little refactor. This should be resolved now.